plugins/koa/index.js

```diff
@@ -0,0 +1,47 @@
+const {transparentGetter, requireFileGetter} = require('@brillout/reconfig/getters');
+
+const packageName = require('./package.json').name;
+
+const serverStartFile = require.resolve('./startServer');
+const koaIntegrationPluginFile = require.resolve('./KoaIntegrationMiddleware');
+
+module.exports = {
+    $name: packageName,
+    $getters: [
+        transparentGetter('serverStartFile'),
+        requireFileGetter('koaIntegrationPluginFile'),
```

Comment:
Let's rename this config to `koaIntegrationFile'

Reply:
Done